README.md

```diff
@@ -14,6 +14,23 @@ For more information about this project of more about NASA's Earth Observing Sys
 
 # 🔨 Development
 
+The Cumulus core repo is a [monorepo](https://en.wikipedia.org/wiki/Monorepo)
```

Comment:
This works - ideally it feels like something we should put (or link)  in the task/package READMEs as well, but I don't feel strongly enough to block the PR on that point so long as this is here - anyone new to Core should be reading this first regardless.